src/main/views/template.njk

```diff
@@ -9,6 +9,32 @@
 
 {% block header %}
 
+{% if isAuthenticated %}
+{{ govukHeader({
```

Comment:
Instead of rendering whole govukHeader in if else, can we just change last navigation item if user is signed in or not.

Reply:
Absolutely - will sort now